dags/solutions_team/configs/tensorflow/common.py

```diff
@@ -139,29 +105,11 @@ def set_up_tensorflow_models(
       "sudo mkdir -p /usr/share/tpu && cd /usr/share/tpu",
       f'if [ ! -d "/usr/share/tpu/models" ]; then sudo git clone -b {models_branch} https://github.com/tensorflow/models.git; fi',
       "pip install -r /usr/share/tpu/models/official/requirements.txt",
-      "pip install tensorflow-recommenders --no-deps",
+      f'if [ ! -d "/usr/share/tpu/recommenders" ]; then sudo git clone -b main https://github.com/tensorflow/recommenders.git; fi',
```

Comment:
do we want to pin the recommenders repository to a specific branch similar to tensorflow models above?

Reply:
> do we want to pin the recommenders repository to a specific branch similar to tensorflow models above?

We're already pinning it to the main branch. `git clone -b main`